### PR TITLE
Fix code bug

### PR DESCRIPTION
**Pull request by MrXhh**
Fix code bug

### CODE REVIEW

**Comment by dougbu** [comment]
@MrXhh could you please file an issue that we can reference in future release notes❔

@mkArtakMSFT we should take this one-line fix because it's a straight-up bug and won't cause breaking changes for our users.

**Review by dougbu — APPROVED** [review]
Thanks @MrXhh❕ We'll get this in once we have an issue to reference in the commit description.

File: src/System.Web.Http/ModelBinding/ErrorParameterBinding.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ public ErrorParameterBinding(HttpParameterDescriptor descriptor, string message)
         {
             if (message == null)
             {
-                throw Error.ArgumentNull(message);
+                throw Error.ArgumentNull("message");
```
Comment (mkArtakMSFT) [inline src/System.Web.Http/ModelBinding/ErrorParameterBinding.cs:23]: Should this be `nameof(message)` instead?
@dougbu  you're probably more familiar with the convention used in this repo.

Comment (dougbu) [reply src/System.Web.Http/ModelBinding/ErrorParameterBinding.cs:23]: We don't use `nameof` at all in this repository. Most of the code here predates C# 6.

**Comment by MrXhh** [comment]
Fix #357

**Pull request merged**